content/en/docs/Writing policies/preconditions.md

```diff
@@ -31,7 +31,7 @@ The following operators are currently supported for precondition evaluation:
 - In
 - NotIn
 
-The set operators, `In` and `NotIn` support a set of strings as the value (e.g. In ["str1", "str2"]). Sets of other types are currently not supported.
+The set operators, `In` and `NotIn` support a set of strings as the value (e.g. In ["str1", "str2"]). They also allow you to specify a set of strings as the key (e.g. ["str1", "str2"] In ["str1", "str2", "str3"]). In this case `In` checks if **all** the strings part of the key are in the value set and `NotIn` checks if **none** of the strings part of the key is in the value set. Sets of other types are currently not supported. 
```

Comment:
"...and `NotIn` checks if **none** of the strings part of the key _are_ in the value set."
Remove space end of line.